collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormsTask.java

```diff
@@ -154,11 +160,11 @@ private String processOneForm(int total, int count, FormDetails fd) throws Cance
                 Timber.i("No Manifest for: %s", fd.formName);
             }
         } catch (TaskCancelledException e) {
-            Timber.i(e);
-            cleanUp(fileResult, e.getFile(), tempMediaPath);
+            Timber.i(e.getMessage());
```

Comment:
`Timber` sorts that out for us so the call on `getMessage()` is redundant in this case. Not a big deal but FYI.

Reply:
Timber logs the whole stack trace. I think we need just the message.

Reply:
Oh, I see. You never want to see the trace when it uses the console Tree. That makes sense. For Firebase it nicely collapses things but since this is info so it will never go there. 👍